### PR TITLE
Regroup output for better debug flow and add missing DBs

### PR DESCRIPTION
**Pull request by kvijai82**
Regroup the output into foundational & cp4waiops section to make it easier to debug issues.  There were a few DBs that were missing as well that were added.

### CODE REVIEW

**Comment by kvijai82** [comment]
What the output looks like:

image

image

**Review by taylormgeorge91 — COMMENTED** [review]
Would it be possible to have it such that the component columns are the same and are a shared printout?

Maybe if some of the custom columns mismatch but can be aliased we can use the following ones with `--no-headers` output to strip that and have htem appear under the same header. i think the main concern would be padding/spacing inconsistencies based on length of values.

At some point we need to circle back to this and start summarizing the output instead of adding more verbosity to it

File: kubectl-plugin/kubectl-waiops [hunk 1]
```diff
@@ -474,9 +527,7 @@ then
         echo ""
         exit 0
     else       
-        printf "${blue}${bold}Hint: log into your cluster with CLUSTER_ADMIN credentials and try again.${normal}
-
-"
+        printf "${blue}${bold}Hint: log into your cluster with CLUSTER_ADMIN credentials and try again.${normal}"
```
Comment (siyer-123) [inline kubectl-plugin/kubectl-waiops:530]: I think we need to add those returns back in. Here's what I see when I run all three commands while not in cluster-admin mode (This line for `status-all` is the one with the formatting issue):

Screen Shot 2022-07-05 at 4 34 27 PM

**Review by siyer-123 — COMMENTED** [review]
Noticed a few minor things when I ran the script on a BVT cluster, but I definitely like the new output flow! Feels more streamlined. Thanks Vijai!

File: kubectl-plugin/kubectl-waiops [hunk 2]
```diff
@@ -122,19 +127,50 @@ then
...
+
         # cp4waiops-eventprocessor EventProcessor status
         INSTANCE_EP=$(oc get eventprocessor cp4waiops-eventprocessor -o custom-columns='KIND:.kind,NAMESPACE:.metadata.namespace,NAME:.metadata.name,VERSION:status.versions.reconciled,STATUS:.status.conditions[?(@.type=="Ready")].status,MESSAGE:.status.conditions[?(@.type=="Ready")].message')
         STATUS_EP=$(oc get eventprocessor cp4waiops-eventprocessor -o jsonpath='{.status.conditions[?(@.type=="Ready")].status}')
         printStatus "$STATUS_EP" "True" "$INSTANCE_EP"
 
+        # Elasticsearch status
+        INSTANCE_ES=$(oc get elasticsearch iaf-system -o custom-columns='KIND:.kind,NAMESPACE:.metadata.namespace,NAME:.metadata.name,READY:.status.conditions[?(@.type=="Ready")].status')
+        STATUS_ES=$(oc get elasticsearch iaf-system -o jsonpath='{.status.conditions[?(@.type=="Ready")].status}')
+        printStatus "$STATUS_ES" "True" "$INSTANCE_ES"
```
[the remark this answers is not in the document]
Comment (kvijai82) [reply kubectl-plugin/kubectl-waiops:145]: Thanks for catching this.  I think on those clusters there is an additional elastic operator that's installed.  I will update to fully qualify it.

**Comment by kvijai82** [comment]
> Would it be possible to have it such that the component columns are the same and are a shared printout?
> 
> Maybe if some of the custom columns mismatch but can be aliased we can use the following ones with `--no-headers` output to strip that and have htem appear under the same header. i think the main concern would be padding/spacing inconsistencies based on length of values.
> 
> At some point we need to circle back to this and start summarizing the output instead of adding more verbosity to it

Agree with the summary part.  Started experimenting with a troubleshoot option that will need a bit more work.

Regarding removing some headers, will experiment a bit to see if I can atleast group some of them together based on the spacing to reduce the real estate of the output a bit.

File: kubectl-plugin/kubectl-waiops [hunk 3]
```diff
@@ -212,46 +242,83 @@ then
...
+
+        # cp4waiops-eventprocessor EventProcessor status
         INSTANCE_EP=$(oc get eventprocessor cp4waiops-eventprocessor -o custom-columns='KIND:.kind,NAMESPACE:.metadata.namespace,NAME:.metadata.name,VERSION:status.versions.reconciled,STATUS:.status.conditions[?(@.type=="Ready")].status,MESSAGE:.status.conditions[?(@.type=="Ready")].message')
         STATUS_EP=$(oc get eventprocessor cp4waiops-eventprocessor -o jsonpath='{.status.conditions[?(@.type=="Ready")].status}')
         printStatus "$STATUS_EP" "True" "$INSTANCE_EP"
 
+        # Elasticsearch status
+        INSTANCE_ES=$(oc get elasticsearch iaf-system -o custom-columns='KIND:.kind,NAMESPACE:.metadata.namespace,NAME:.metadata.name,READY:.status.conditions[?(@.type=="Ready")].status')
+        STATUS_ES=$(oc get elasticsearch iaf-system -o jsonpath='{.status.conditions[?(@.type=="Ready")].status}')
+        printStatus "$STATUS_ES" "True" "$INSTANCE_ES"
```
[the remark this answers is not in the document]
Comment (siyer-123) [reply kubectl-plugin/kubectl-waiops]: Resolved in latest commit.

File: kubectl-plugin/kubectl-waiops (hunk 2 above)
[the remark this answers is not in the document]
Comment (siyer-123) [reply kubectl-plugin/kubectl-waiops]: Resolved in latest commit.

File: kubectl-plugin/kubectl-waiops (hunk 1 above)
Comment (siyer-123) [reply kubectl-plugin/kubectl-waiops:538]: @kvijai82 wondering if we added these spaces back in.

**Comment by siyer-123** [comment]
Just ran the updated script on a few clusters (my v3.4 test cluster + a BVT on `main`) and noticed this error message pop up in the OperandRequests section:

```
NAMESPACE   NAME         PHASE     CREATED AT
katamari    iaf-system   Running   2022-07-22T13:48:23Z

Error from server (NotFound): operandrequests.operator.ibm.com "iaf-system-common-service" not found
Error from server (NotFound): operandrequests.operator.ibm.com "iaf-system-common-service" not found


NAMESPACE   NAME                   PHASE     CREATED AT
katamari    ibm-aiops-ai-manager   Running   2022-07-22T13:43:53Z
```

Other than this, everything else looks good.

**Pull request closed without merging**